docs/product-installation-guide/src/main/asciidoc/git-import-proc.adoc

```diff
@@ -0,0 +1,16 @@
+[id='git-import-proc']
+
+= Importing projects from Git repositories
+Git is a distributed version control system. It implements revisions as commit objects. When you commit your changes into a repository, a new commit object in the Git repository is created. When you create a project in {CENTRAL} it is added to the Git repository connected to {CENTRAL}.
+
+If you have projects in other Git repositories, you can import them into {CENTRAL} spaces.
+
+.Prerequisites
+* {PRODUCT} projects in an external Git repository
+
+.Procedure
+. In {CENTRAL}, click *Menu* -> *Design* -> *Projects*.
+. Select or create the space into which you want to import the projects. The default space is *myteam*.
+. Click the three verticle dots on the right side of the screen and select *Import Project*.
+. In the Import Project window, enter the URL and credentials for the Git repository that contains the projects that you want to import and click *Import*. The projects are added to the current space.
```

Comment:
In the **Import Project** window. Per previous style decision to bold UI elements in general.